packages/lib/src/vis/shared/Html.tsx

```diff
@@ -8,14 +8,15 @@ import {
   useCallback,
   useEffect,
 } from 'react';
-import type { HTMLAttributes } from 'react';
+import type { HTMLAttributes, RefCallback } from 'react';
 import ReactDOM from 'react-dom';
 import type { Group } from 'three';
 import { Vector3 } from 'three';
 
 import { useAxisSystemContext } from './AxisSystemContext';
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
+  refCallback?: RefCallback<HTMLDivElement>;
```

Comment:
I need to pass a `ref` to `Html` so I can get a reference to the DOM element of the floating toolbar.

However, the vis canvas is rendered dynamically into the DOM, so the ref is `null` on first render. `FloatingControl` needs to know when this ref has been set, so the only way to do that is to use a ref callback. To simplify typings, I decided to use a prop, as `forwardRef` allows for a lot more types of ref to be passed. This technique is mentioned here: https://reactjs.org/docs/refs-and-the-dom.html#exposing-dom-refs-to-parent-components

- About callback refs: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs
- About re-rendering when ref gets set: https://reactjs.org/docs/hooks-reference.html#useref (cf. note at bottom of section)